packages/image_picker/image_picker_platform_interface/test/new_method_channel_image_picker_test.dart

```diff
@@ -929,6 +929,40 @@ void main() {
         expect(response.file!.path, '/example/path');
       });
 
+      test('getLostData get success response with pathList', () async {
```

Comment:
Similarly
```suggestion
      test('getLostData should successfully retrieve multiple files', () async {
```
Additionally maybe update the `pathList` property to return more than 1 path as well, since this specifically applies to selecting multiple images?